patches/minecraft/net/minecraft/client/model/PiglinModel.java.patch

```diff
@@ -0,0 +1,10 @@
+--- a/net/minecraft/client/model/PiglinModel.java
++++ b/net/minecraft/client/model/PiglinModel.java
+@@ -81,6 +_,7 @@
+             }
+          }
+       } else if (p_103366_.m_6095_() == EntityType.f_20531_) {
++         if (!net.minecraftforge.client.ForgeHooksClient.hasForgeArmPose(p_103366_))
```

Comment:
This is weird since this only handles zombie piglins and not regular piglins, though I understand that's due to the different uses of poses. Though, I think you could cover both simply by putting this statement above the class check for piglin.